src/main/java/core/PomDependencyReader.java

```diff
@@ -0,0 +1,54 @@
+package core;
+
+import org.apache.maven.model.Dependency;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.*;
+import java.util.*;
+
+/**
+ * Makes a list with all dependencies from the pom.xml
+ */
+public class PomDependencyReader {
+    private static final Logger LOGGER = LoggerFactory.getLogger(PomDependencyReader.class);
+    private String path;
+    /**
+     * Init PomDependencyReader
+     * @param path Path describes where the pom.xml file is
+     * @throws FileNotFoundException
+     */
+    public PomDependencyReader(String path) throws FileNotFoundException {
```

Comment:
File Object wäre schöner, muss aber nicht
Schmeisst doch gar keine FileNotFoundException
```suggestion
    public PomDependencyReader(String path) {
```

Reply:
geändert